### PR TITLE
Add session keys feature flag

### PR DESCRIPTION
**Pull request by raykyri**
Closes: #5114

- Puts server-side enforcement of session keys behind a feature flag (ENFORCE_SESSION_KEYS=true)
- Still tries to generate a session key for Magic users, but fails quietly if unsuccessful
- Fixes ES module import error -- this was already patched on master
- Fixes a small UI issue with web wallet listing

Test plan is the same as rest of session keys.

### CODE REVIEW

File: packages/commonwealth/client/scripts/controllers/server/sessions.ts [hunk 1]
```diff
@@ -212,18 +212,22 @@ class SessionsController {
       address
     );
 
-    // Get a new session signature.
+    // // Get a new session signature.
+    // if (!hasAuthenticatedSession) {
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/controllers/server/sessions.ts:216]: Can we add something searchable here -- or alternatively, gate this block with the session keys flag?

File: packages/commonwealth/client/scripts/views/components/Header/UserDropdown/UserDropdown.tsx [hunk 2]
```diff
@@ -84,20 +84,21 @@ const UserDropdown = () => {
...
             hasJoinedCommunity={isActive}
             address={account.address}
           />
         ),
         onClick: async () => {
-          if (signed) {
-            return await setActiveAccount(account);
-          }
+          return await setActiveAccount(account);
+          // if (signed) {
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/components/Header/UserDropdown/UserDropdown.tsx:94]: Same comment as above -- just want to make sure we don't miss it in fast follows.

**Review by jnaviask — COMMENTED** [review]
Generally looks good to move onto QA. Would like a few small things beforehand.

Also -- what config changes do we need magic-side?

File: packages/commonwealth/client/scripts/controllers/app/login.ts [hunk 3]
```diff
@@ -436,81 +436,83 @@ export async function handleSocialLoginCallback({
...
-      magicAddress
-    );
-  } else {
-    const { Web3Provider } = await import('@ethersproject/providers');
-    const { utils } = await import('ethers');
+      // Request the cosmos chain ID, since this is used by Magic to generate
+      // the signed message. The API is already used by the Magic iframe,
+      // but they don't expose the results.
+      const nodeInfo = await $.get(
+        `${document.location.origin}/magicCosmosAPI/${desiredChain.id}/node_info`
```
[the remark this answers is not in the document]
Comment (raykyri) [reply packages/commonwealth/client/scripts/controllers/app/login.ts:456]: We might still need it in the future. Magic resolved the CSP issue but they still have a restrictive CORS policy, which might require us to use a local proxy for /status and /node_info requests unless they change that as well.

**Review by jnaviask — APPROVED** [review]
Seems fine -- no obvious major issues. Good to get this on QA.

**Pull request merged**